dist/cargo_test.sh

```diff
@@ -7,12 +7,14 @@ cargo_test_flags["cincinnati"]="--features test-net"
 cargo_test_flags["commons"]=""
 cargo_test_flags["graph-builder"]="--features test-net"
 cargo_test_flags["policy-engine"]=""
+cargo_test_flags["prometheus-query"]=""
 cargo_test_flags["quay"]="--features test-net"
 
 if [[ -n "${CINCINNATI_TEST_CREDENTIALS_PATH}" && -n "${CINCINNATI_TEST_QUAY_API_TOKEN_PATH}" ]]; then
     echo Secrets available, enabling private tests...
     cargo_test_flags["cincinnati"]+=",test-net-private"
     cargo_test_flags["graph-builder"]+=",test-net-private"
+    cargo_test_flags["prometheus-query"]="--features test-net-private"
```

Comment:
The `test-net-private` feature is gone as of https://github.com/openshift/cincinnati/pull/249/commits/21a18bc329384ad8be2906f8506dc06ae2a89ac2 so it can be removed here.

```suggestion
```

Reply:
Removed